custom-events-generated/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "custom-events-generated"
+version = "0.1.0"
+authors = ["Orhun Parmaksız <orhunparmaksiz@gmail.com>"]
+license = "MIT"
+edition = "2021"
```

Comment:
We can probably bump this to 2024 for apps without any problems

Reply:
https://github.com/ratatui/templates/issues/90